### PR TITLE
Riley/c 2496 deploy webpack bundles to s3 and put

### PR DESCRIPTION
**Pull request by rileylnapier**
## Description

deploy to s3 bucket with latest.js

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> `Fix [#1]()`

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -11,6 +11,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - name: Set Env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
```
Comment (rileylnapier) [inline .github/workflows/release.yml:15]: planning on using this later for versioning

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -21,3 +23,12 @@ jobs:
       - run: yarn lerna:publish
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
+      - run: yarn build:components
+      - uses: jakejarvis/s3-sync-action@master
+        with:
+          args: --acl public-read --follow-symlinks --delete
+        env:
+          AWS_S3_BUCKET: "courier-components-xvdza5"
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
```
Comment (rileylnapier) [inline .github/workflows/release.yml:32]: already created and added

File: packages/components/webpack.config.js [hunk 3]
```diff
@@ -8,13 +8,19 @@ module.exports = {
   mode: "development",
   entry: path.resolve(__dirname, "./src/index.tsx"),
   output: {
+    publicPath:
+      process.env.NODE_ENV === "production"
+        ? "https://courier-components-xvdza5.s3.amazonaws.com/"
+        : undefined,
+    filename: "latest.js",
```
Comment (rileylnapier) [inline packages/components/webpack.config.js:15]: maybe will use version as the filename in the future... but for now latest until we have a discussion on versioning

File: .github/workflows/release.yml (hunk 1 above)
Comment (cgradwohl) [reply .github/workflows/release.yml:15]: BASH FTW

**Review by cgradwohl — APPROVED** [review]

**Review by cgradwohl — APPROVED** [review]

**Pull request merged**